tests/bluetooth/controller/mock_ctrl/src/mayfly.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2016-2017 Nordic Semiconductor ASA
+ * Copyright (c) 2016 Vinayak Kariappa Chettimada
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ztest.h>
+#include "memq.h"
+
+#define MAYFLY_CALL_ID_0			 0
+#define MAYFLY_CALL_ID_1			 1
+#define MAYFLY_CALL_ID_2			 2
+#define MAYFLY_CALL_ID_PROGRAM 3
+#define MAYFLY_CALLER_COUNT		4
+#define MAYFLY_CALLEE_COUNT		4
+
+struct mayfly {
+	u8_t volatile _req;
+	u8_t _ack;
+	memq_link_t *_link;
+	void *param;
+	void (*fp)(void *);
+};
+
+void mayfly_init(void)
+{
+
```

Comment:
Unnecessary empty line. I don't think we have any official style for empty functions, but you're being inconsistent even with yourself here. In other places you don't have an empty line. Stick to just one style.